server.go

```diff
@@ -12,26 +13,32 @@ import (
 
 func main() {
 	e := echo.New()
-	e.GET("/", func(c echo.Context) error {
-
-		data := getData()
-		return c.String(http.StatusOK, data)
-	})
+	e.GET("/", defaultRoute)
```

Comment:
clean up routing by separating the route logic into a dedicated function (and passing the function as argument)